mautic/mautic
# Contact ID column to plugin integration mapping

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | Y
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/pull/5310
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

This bugfix/feature has been sponsored by @Webmecanik

[//]: # ( Required: )
#### Description:

This PR add the contact ID in plugin mapping. That can help to send mautic contact ID into a CRM for instance.  This PR comes in favor of https://github.com/mautic/mautic/pull/5310 (@npracht  please close old PR )

#### Steps to test this PR:

1. Apply the PR
2. Configure a plugin
3. Go to contact field mapping
4. Notice that the Mautic contact ID is available and sync to integration
